test/ir/inference/test_trt_convert_one_hot.py

```diff
@@ -83,7 +84,7 @@ def generate_depth(dims, batch):
 
     def sample_predictor_configs(
         self, program_config
-    ) -> (paddle_infer.Config, List[int], float):
+    ) -> (paddle_infer.Config, list[int], float):
```

Comment:
```suggestion
    ) -> tuple[paddle_infer.Config, list[int], float]:
```

其他的小问题可以不改，但这里是个浅显的错误，不符合类型提示的语法，需要修改下

其他的也一样，就不一一 comment 了